.github/workflows/quality.yaml

```diff
@@ -24,10 +24,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python setup.py egg_info
-          sed '/^$/q' src/lighteval.egg-info/requires.txt > src/lighteval.egg-info/requires_lite.txt
-          python -m pip install ruff -c src/lighteval.egg-info/requires_lite.txt
-          rm -rf src/lighteval.egg-info
+          python -m pip install ".[quality]"
```

Comment:
I wasn't sure why you had the `sed` stuff above, but this seems simpler?

Reply:
Because we did not want to install the full dependencies just to launch the ruff styling :) 
So we select the correct ruff version from the dep (we have to extract them from the pytoml, then clean the requirements file created), and install only this

Reply:
Ah makes sense! I will revert that change :)

Reply:
Reverting the change causes some peculiar `ruff` errors, where it seems to install the wrong version and thus says various files should be formatted 🤔 

Revisiting the above point about keeping it light - is it really a bottleneck in the CI? We don't really worry about this in the other HF libs, and some of them (like `accelerate`) also have some heavy deps like `torch`

Reply:
It just seems like a waste of compute time ^^, but I'm OK with using your version since it's going to be more legible.